build/common.sh

```diff
@@ -713,7 +714,7 @@ function kube::build::copy_output() {
   kube::build::start_rsyncd_container
 
   local rsync_extra=""
-  if (( ${KUBE_VERBOSE} >= 6 )); then
+  if (( KUBE_VERBOSE >= 6 )); then
     rsync_extra="-iv"
```

Comment:
@BenTheElder 
I am not able to figure out where the variable rsync_extra is used.
there is still one warning message as shown below
```bash
SC2034: rsync_extra appears unused. Verify use (or export if used externally).
```

Reply:
Not sure if anything is currently, we might have to look into the history of this script:
https://cs.k8s.io/?q=rsync_extra&i=nope&files=&repos=

Reply:
Hey @BenTheElder 
After going through 
https://github.com/kubernetes/kubernetes/commit/dc586ea8f799c48a916d9bac4db4bc8cb8fa297f#diff-0ee501c7b4b1ab61678579826c6d6534
I came to know that all the logic related to rsync_extra have been moved to the function kube::build::rsync() and these lines can removed safely.

Reply:
So can i remove these lines.

Reply:
Yes that SGTM, thank you!

Reply:
Done